src/main/java/step2/domain/WinningLotto.java

```diff
@@ -0,0 +1,56 @@
+package step2.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+public class WinningLotto {
+
+    private static final int LOTTO_NUM_COUNT = 6;
+
+    private final Set<LottoNumber> winningLottoNumbers;
+    private final LottoNumber bonusNumber;
+
+    private WinningLotto(Set<LottoNumber> winningLottoNumbers, LottoNumber bonusNumber) {
+        if (winningLottoNumbers.size() != LOTTO_NUM_COUNT) {
+            throw new IllegalArgumentException("로또 번호는 총" + LOTTO_NUM_COUNT + "개 이어야 합니다.");
+        }
```

Comment:
해당 validation은 중복되서 수행되고 있네요! 이미 Lotto객체에서 수행하고 있기에
Set 자료구조를 파라미터로 던지는것보다 Lotto 객체로 변환해서 던지면 어떤 장점들이 있을까요~?
```suggestion
    private WinningLotto(Lotto winningLottoNumbers, LottoNumber bonusNumber) {
```